big-dipper/imports/api/recipes/server/methods.js

```diff
@@ -56,7 +56,7 @@ Meteor.methods({
           let recipe = recipes[i];
           let deeplink =
             Meteor.settings.public.baseURL +
-            "?action=purchase_nft&recipe_id=" +
+            "?recipe_id=" +
```

Comment:
when we are building the deep link, do we need to sanitize the URL, any concerns?